services/hierarchy/src/test/kotlin/PackageServiceTest.kt

```diff
@@ -0,0 +1,510 @@
+                                "https://example.com/example-1.0-sources.jar",
+                                "sha1:source",
+                                "SHA-1"
+                            ),
+                            vcs = VcsInfo(
+                                RepositoryType(""),
+                                "https://example.com/git",
+                                "revision1",
+                                "path"
+                            ),
+                            vcsProcessed = VcsInfo(
+                                RepositoryType("GIT"),
+                                "https://example.com/git",
+                                "revision2",
+                                "path"
+                            ),
+                            isMetadataOnly = false,
+                            isModified = false
+                        ),
+                        Package(
+                            Identifier("Maven", "com.example", "example2", "1.0"),
+                            purl = "pkg:maven/com.example/example2@1.0",
+                            cpe = null,
+                            authors = emptySet(),
+                            declaredLicenses = emptySet(),
+                            ProcessedDeclaredLicense(
+                                spdxExpression = "Expression",
+                                mappedLicenses = emptyMap(),
+                                unmappedLicenses = emptySet()
+                            ),
+                            description = "Another example package",
+                            homepageUrl = "https://example.com",
+                            binaryArtifact = RemoteArtifact(
+                                "https://example.com/example2-1.0.jar",
+                                "sha1:binary",
+                                "SHA-1"
+                            ),
+                            sourceArtifact = RemoteArtifact(
+                                "https://example.com/example2-1.0-sources.jar",
+                                "sha1:source",
+                                "SHA-1"
+                            ),
+                            vcs = VcsInfo(
+                                RepositoryType("GIT"),
+                                "https://example.com/git",
+                                "revision",
+                                "path"
+                            ),
+                            vcsProcessed = VcsInfo(
+                                RepositoryType("GIT"),
+                                "https://example.com/git",
+                                "revision",
+                                "path"
+                            ),
+                            isMetadataOnly = false,
+                            isModified = false
+                        )
+                    ),
+                ).id
+
+                val results = service.listForOrtRunId(ortRunId).data
+
```

Comment:
You could define variables for the two packages used above and the just compare the result with `service.listForOrtRunId(ortRunId).data should containExactlyInAnyOrder(pkg1, pkg2)`. It will be harder to read the diff if the test fails, but it requires a lot less code.